index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/style/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="flex justify-between" id="header">
+    <p class="self-center" id="myLogo">My Logo</p>
```

Comment:
- [ ] I would kindly suggest you wrap your logo inside of an `anchor tag`.
     >The logo (if the design has one) is on the left and has a link (`a` tag). This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

    E.G: 
    ```HTML
      <p class="self-center" id="myLogo"><a href="#">My Logo</a></p>
    ```